scenarios/arcaflow/sysbench/input.yaml

```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
+sysbench_cpumaxprime: 12000
```

Comment:
We need to expose two more options to the user: 1) Node to target on OpenShift/Kubernetes env 2) Percent CPU to hog given that we might not know exact number of cores available. Thoughts?

Reply:
Options exposed on the arca plugin side